Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12-slim
+LABEL maintainer="mxbely98@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
Consider adding a `COPY` command before installing dependencies to copy only the `requirements.txt` file first. This allows Docker to cache the pip install step, which can speed up subsequent builds if `requirements.txt` hasn't changed.